spec/definitions/ExportSchedule.yaml

```diff
@@ -0,0 +1,103 @@
+type: object
+required:
+  - format
+  - name
+  - recurringInstruction
+  - scheduledStartDate
+properties:
+  id:
+    description: The export identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  status:
+    description: Status of export request
+    readOnly: true
+    type: string
+    enum:
+      - "pending"
+      - "processing"
+      - "completed"
+  name:
+    description: Export name
+    type: string
+  resource:
```

Comment:
I know `/exports/transactions` has `resource`, but it seems redundant to have it in the uri and in the response.